package.json

```
@@ -74,6 +71,8 @@
   },
   "scripts": {
     "prepare": "node --eval \"1+1; // help yarn users not install the whole repo; https://github.com/yarnpkg/yarn/issues/2822#issuecomment-847360267\"",
+    "prepack": "cp README.md README-orig.md && cp npm-README.md README.md",
+    "postpack": "test -f README-orig.md && cp README-orig.md README.md && rm README-orig.md || true",
```

Is this the logic you're looking for?

```suggestion
    "postpack": "test -f README-orig.md && cp README-orig.md README.md && (rm README-orig.md || true)",
```

---

Actually, I think I can take the `true` out.